src/test/java/ca/wolfram/beta/symath/NodeTypeTest.java

```diff
@@ -0,0 +1,35 @@
+package ca.wolfram.beta.symath;
+
+import ca.wolfram.beta.symath.base.BaseNode;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * Created by Allan Wang on 2017-05-09.
+ */
+public class NodeTypeTest {
+
+    @Test
+    public void names() {
+        assertEquals("PowerNode", NodeType.POWER.getName());
+        assertEquals("MathConstantNode", NodeType.MATH_CONSTANT.getName());
+    }
+
+    @Test
+    public void validate() {
+        List<MathNode> list = new ArrayList<>();
+        list.add(BaseNode.create(0));
+        NodeType.ADD.validateListSize(list); //no error should be thrown
```

Comment:
I think it would be better to catch the error then fail the test. then just let it throw an exception